ebu_tt_live/node/element_remover.py

```diff
@@ -0,0 +1,131 @@
+from .base import AbstractCombinedNode
+from ebu_tt_live.documents.ebutt3 import EBUTT3Document
+from ebu_tt_live.errors import UnexpectedSequenceIdentifierError
+from pyxb.binding.basis import ElementContent, \
+    complexTypeDefinition, simpleTypeDefinition
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class ElementRemoverNode(AbstractCombinedNode):
+    """
+    Removes unwanted elements from a document.
+
+    Matches elements by local names only. Restricted to
+    XML elements that have bindings; non-bound elements
+    are left in place.
+    """
+
+    _expects = EBUTT3Document
+    _provides = EBUTT3Document
+
+    _remove_list = []
+    _sequence_identifier = None
+    _last_sequence_number = None
+
+    def __init__(self,
+                 node_id,
+                 sequence_identifier,
+                 consumer_carriage=None,
+                 producer_carriage=None,
+                 remove_list=''):
+        super(ElementRemoverNode, self).__init__(
+            node_id=node_id,
+            producer_carriage=producer_carriage,
+            consumer_carriage=consumer_carriage
+        )
+
+        self.remove(remove_list)
```

Comment:
Seems like an odd pattern to have a function called `remove` that updates the list of elements to remove. Should this just be part of the constructor?

Reply:
It's in the constructor too, but it was helpful to be able to set the remove list afterwards. Would it help to change the name to `set_remove_list` or something like that?

Reply:
After discussion, I will change this to a private `_set_remove_list()` method and modify the tests to set the remove list only in the constructor.

Reply:
Done in cb4e4cc